lib/WindowManager.vala

```diff
@@ -66,7 +66,14 @@ namespace Gala {
          * Defaults to blocking all.
          * @see KeybindingFilter
          */
-        public KeybindingFilter? keybinding_filter { get; owned set; default = () => true; }
+        private KeybindingFilter? _keybinding_filter = () => true;
+        public KeybindingFilter? get_keybinding_filter () {
+            return _keybinding_filter;
+        }
+
+        public void set_keybinding_filter (KeybindingFilter? filter) {
+            _keybinding_filter = filter;
```

Comment:
```suggestion
            _keybinding_filter =  (owned) filter;
```

Reply:
This change gives a compile error:
`../lib/WindowManager.vala:75.35-75.48: error: No reference to be transferred`